### PR TITLE
6199 adjust alignment on the Order Cycle component on ipad

### LINKED ISSUE #6199: [mobile ux] Correct alignment on "ready for" field on Ipad
## Description

On an iPad, the ready for field does not show up vertically centered.

A 1em/16px padding you be introduce above the dropdown.

## Steps to Reproduce
1. Go visit an open shop with an Ipad

## Animated Gif/Screenshot

Landscape

image

Portrait

image

## Workaround
None

## Severity
bug-s4: it's annoying, but you can use it

## Your Environment
* Version used: v3.2.10
* Browser name and version: Firefox, Chrome & Safari
* Operating System and version (desktop or mobile): Tablet

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?
Closes #6199 
Adjust vertical alignment of the component "Order Cycle" (and specially the "Ready for" field) on a shop on iPad device.

#### What should we test?
On an iPad (and other devices, depending on screen width), the ready for field must be vertically centered.

#### Release notes
Fix vertical alignment issue on order cycle on a shop.

Changelog Category: User facing changes

### CODE REVIEW

File: app/views/enterprises/_change_order_cycle.html.haml [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
-        = t :enterprises_currently_open
-    %div{"ng-if" => "!OrderCycle.selected()"}
-      = t :enterprises_choose
+  .select-and-closing-container
+    %closing
+      %div{"ng-if" => "OrderCycle.selected()"}
+        %div{"ng-if" => "closesInLessThan3Months()"}
+          = t :enterprises_next_closing
+          %strong {{ OrderCycle.orders_close_at() | date_in_words }}
+        %div{"ng-if" => "!closesInLessThan3Months()"}
```
[the remark this answers is not in the document]
Comment (sauloperez) [reply app/views/enterprises/_change_order_cycle.html.haml:8]: I opened https://github.com/openfoodfoundation/openfoodnetwork/pull/6630 to get rid of this.

**Review by sauloperez — APPROVED** [review]

**Review by andrewpbrett — APPROVED** [review]
The code looks sensible, but when I checked out this branch locally and opened it in responsive mode in Safari, it doesn't look like it's vertically centered yet:
Screen Shot 2021-01-08 at 1 54 15 PM

If you're seeing something different I think it's fine to move it to test ready and I'll trust Filipe'e process instead of mine :)

**Comment by jibees** [comment]
Thanks @andrewpbrett for testing it!
I've tested on my side, and it's vertically centered... 
Capture d’écran 2021-01-10 à 09 21 34
I've previously also tested with browserstack with success.
The multi-devices joy !

**Comment by andrewpbrett** [comment]
I'm guessing it's PEBKAC on my end then... moving to test ready.

**Comment by sauloperez** [comment]
lol @andrewpbrett I didn't know that acronym. Or that that expression had an acronym :joy:

**Comment by RachL** [comment]
I didn't know the English version 🤣 In French we say ICC (interface chaise-clavier => chair.keyboard interface)

**Comment by filipefurtad0** [comment]
Hey @jibees ,

I was aiming to reproduce this in our staging servers, but did not manage to observe this, except in staging-AU which was probably not updated with this release:

image

This is the same device, showing the production server from Katuma:

image


However, I did notice that the closing order cycle message was always displayed over the "ready for" details. This is on Safari on iPad 5th:

Before staging your PR:
image

After staging we can see that this is now displayed next to the OC details, and looks great :+1: 
image

Safari on iPad Pro 14
image

Safari on iPad Mini 4
image

I checked this as well, Safari on iPhones 12 (regression test):

before staging:
image


after staging:
image


There is some difference above - we can see that after staging the OC closure is not centred on top of the "ready for" details. But I think this is ok. I'd need a second opinion, on whether this would need to be centred - pinging @openfoodfoundation/train-drivers-product-owners - if the feedback goes on that direction, them we can open a separate issue, I think.

Moving to ready to go, thanks JB.

**Comment by RachL** [comment]
hmmm so fixing iPad has consequences on iPhones... Maybe worth checking Android devices as well? 🤔 if this is too complex to fix we can leave it for now, the PR already improves a lot the situation.

**Pull request merged**